databricks_cli/pipelines/api.py

```diff
@@ -72,27 +124,30 @@ def _get_hashed_path(path):
             raise RuntimeError('Error \'{}\' while processing {}'.format(e, path))
 
         file_hash = hash_buffer.hexdigest()
-        path = 'dbfs:/pipelines/code/{}{}'.format(file_hash, os.path.splitext(path)[1])
+        path = '{}{}{}'.format(base_pipelines_dir, file_hash, os.path.splitext(path)[1])
         return path
 
-    def _get_remote_mappings(self, local_lib_objects):
-        return list(map(lambda llo: LibraryObject(llo.lib_type, self._get_hashed_path(llo.path)),
-                        local_lib_objects))
-
     def _get_files_to_upload(self, local_lib_objects, remote_lib_objects):
+        """
+        Returns a Local/Remote pair for every file that needs to be uploaded to dbfs
```

Comment:
I'd mention here that files are stored under base_pipeline_dir and named after the hashes of their contents, and we only upload files that don't already exist there.